src/verifiable-presentation.js

```diff
@@ -80,13 +84,28 @@ class VerifiablePresentation {
     return vp;
   }
 
+  /**
+   * Sets the context to the given value, overrding all others
+   * @param {string|object} context - Context to assign
+   * @returns {VerifiablePresentation}
+   */
+  setContext(context) {
```

Comment:
I had raised this point in earlier reviews as well, let's not duplicate the functions. In this case `setContext` has same logic as in `VerifiableCredential`. Same with `addContext`

Reply:
It does have same logic, but are you proposing to write and export a shared function to handle both cases in these classes? Dont think there's a need for such simple logic, if it was more complex or more lines I'd be more inclined to reflector that way

Reply:
Ok